src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
```

Comment:
As to passing in arguments you could use a PropertyFactory (a function/class that generates other classes)

```

import {StandardProperty, Railroad, Utilty, etc. }

// This would replace that landon obj:
const PROPERTY_TYPES = {
     'standard': StandardProperty,
     'railroad': Railroad,
     'utility': Utility,
/// etc.
};

class BaseProperty {
    constructor(obj) {
        for (let key in object) {
            // All subclasses of BaseProperty will now have all the original 
            //    attributes from the json. 
            this[key] = obj[key];
        }
    }
};

function getMeMyProperty(objFromJson) {
        const Cls = PROPERTY_TYPES[objFromJson.propertyType];

        return new Cls(objFromJson);
}

```